### PR TITLE
fix: AOT compilation

### PR DESCRIPTION
**Pull request by yairtawil**

### CODE REVIEW

File: src/app/@ansyn/ansyn/app-effects/effects/map/visualizers.app.effects.spec.ts [hunk 1]
```diff
@@ -195,16 +195,6 @@ describe('VisualizersAppEffects', () => {
 		expect(visualizersAppEffects.shouldDrawOverlaysOnMap$).toBeObservable(expectedResults);
 	});
 
-	it('drawOverlaysOnMap$ should call drawOverlayOnMap() for each map(from selected case)', () => {
```
Comment (liorze) [inline src/app/@ansyn/ansyn/app-effects/effects/map/visualizers.app.effects.spec.ts]: Why did you remove the test?

File: src/app/@ansyn/plugins/openlayers/open-layer-visualizers/entities-visualizer.ts [hunk 2]
```diff
@@ -306,4 +306,13 @@ export abstract class EntitiesVisualizer extends BaseImageryVisualizer {
 		}
 	}
 
+	geometryToEntity(id, geometry): IVisualizerEntity {
+		const featureJson: GeoJSON.Feature<any> = {
```
Comment (liorze) [inline src/app/@ansyn/plugins/openlayers/open-layer-visualizers/entities-visualizer.ts]: You could use turf here.

File: src/app/@ansyn/ansyn/app-effects/effects/map/visualizers.app.effects.spec.ts (hunk 1 above)
Comment (yairtawil) [reply src/app/@ansyn/ansyn/app-effects/effects/map/visualizers.app.effects.spec.ts]: No drawOverlaysOnMap$   effect anymore

File: src/app/@ansyn/plugins/openlayers/open-layer-visualizers/entities-visualizer.ts (hunk 2 above)
Comment (yairtawil) [reply src/app/@ansyn/plugins/openlayers/open-layer-visualizers/entities-visualizer.ts]: How?
